dev-go/licenseclassifier/licenseclassifier-0_pre20200426.ebuild

```diff
@@ -0,0 +1,81 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+EGO_PN=github.com/google/${PN}
+EGIT_REPO_URI="https://${EGO_PN}.git"
+
+inherit go-module
+
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+
+	src_unpack() {
+		git-r3_src_unpack
+		go-module_live_vendor
+	}
+else
+	KEYWORDS="~amd64"
+	EGIT_COMMIT="879cb1424de0ab6dbb3d7a0788a0e40c2515a1b7"
+	SRC_URI="https://${EGO_PN}/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+
+	EGO_SUM=(
+		"github.com/davecgh/go-spew v1.1.0"
+		"github.com/google/go-cmp v0.2.0"
+		"github.com/pmezard/go-difflib v1.0.0"
+		"github.com/sergi/go-diff v1.0.0"
+		"github.com/stretchr/testify v1.3.0"
+
+		"github.com/davecgh/go-spew v1.1.0/go.mod"
+		"github.com/google/go-cmp v0.2.0/go.mod"
+		"github.com/pmezard/go-difflib v1.0.0/go.mod"
+		"github.com/sergi/go-diff v1.0.0/go.mod"
+		"github.com/stretchr/objx v0.1.0/go.mod"
+		"github.com/stretchr/testify v1.3.0/go.mod"
+	)
+
+	S="${WORKDIR}/${PN}-${EGIT_COMMIT}"
+fi
+
+go-module_set_globals
+SRC_URI+=" ${EGO_SUM_SRC_URI}"
+
+DESCRIPTION="A library and set of tools that can analyze text to determine what type of license it contains"
+HOMEPAGE="https://github.com/google/licenseclassifier"
+LICENSE="Apache-2.0 MIT"
+SLOT="0"
+
+src_prepare() {
+	eapply_user
+	rm -v licenses/*db
```

Comment:
`|| die`. Why -v?

Reply:
`-v`: I just tend to do everything verbose especially with 9999 ebuilds,  I've removed it in snapshot and kept it in 9999

I don't think it's a problem if they decide to get rid of that file, so now it's `|| die` in snapshot and `|| :` in live ebuild